eng/Version.Details.xml

```diff
@@ -6,9 +6,17 @@
     </Dependency>
   </ProductDependencies>
   <ToolsetDependencies>
-    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="1.0.0-beta.18516.5">
+    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="1.0.0-beta.19309.1">
       <Uri>https://github.com/dotnet/arcade</Uri>
-      <Sha>3f6ded0d4d606ed995f275a47c154ca6a22e8c7b</Sha>
+      <Sha>45361625125cdd3119050a852bd73163ca4b73f4</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.DotNet.Build.Tasks.Feed" Version="2.2.0-beta.19309.1">
```

Comment:
I would dismiss the SignTool and Task Feed changes. We don't explicitly use them I believe. Let Arcade use them and keep them in sync.

Reply:
Does it hurt anything to leave them? I started another official build and what to just get this PR in.

Reply:
These don't affect product build. They only affect automatic intake of dependencies.